### ISSUE TITLE
Error subscribing to Characteristic

### ISSUE THREAD

**Issue by arafi123** [adabru/BleWinrtDll#58]
Hello,
I am trying to connect to BLE Weight scale (Service 0x181D). When I subscribe to characteristic , I get following error:

`BleWinrtDll.cpp:501 SubscribeCharacteristicAsync catch: The attribute requires authentication before it can be read or written.`

How can I fix this ? What needs to added as authentication parameter in code so that the data can be read from the device ?

I look forward to your support.

Kind regards

**Comment by adabru** [comment]
Hi @arafi123 , maybe you need to pair the device. Maybe related: https://github.com/adabru/BleWinrtDll/issues/7

**Comment by arafi123** [comment]
You mean pair it with PC ? For this BLE Weight scale, I get the same authentication error also with the Microsot BLE sample so I assume may be the device is encrypted or something.

**Comment by adabru** [comment]
Yes, I meant pairing it with the PC. Did you try that? I think you can do it via Windows Settings, I'm not  sure.
